conf/application.conf

```diff
@@ -35,6 +35,11 @@ play.i18n.langs = [ "en" ]
 # db.default.username=sa
 # db.default.password=""
 
+db.default.driver=org.postgresql.Driver
+db.default.url="jdbc:postgresql://localhost/test"
+db.default.username="postgres"
+db.default.password="password"
```

Comment:
Keep this file in the repo for now. Later will add to gitignore and provide better password